packages/juce-blueprint/package.json

```diff
@@ -2,48 +2,37 @@
   "name": "juce-blueprint",
   "version": "0.1.0",
   "description": "BlueprintPlugin user interface in React.",
-  "main": "./cjs/juce-blueprint.js",
-  "types": "./cjs/juce-blueprint.d.ts",
-  "scripts": {
-    "build": "npm run build:types && npm run build:js",
-    "build:types": "tsc --emitDeclarationOnly --outFile ./cjs/juce-blueprint.d.ts",
-    "build:js": "babel esm --out-dir cjs --extensions \".js,.ts\"",
-    "init": "node scripts/init.js"
-  },
+  "main": "./dist/index.js",
```

Comment:
Is dist the best option here? Should it be `umd` ?

Reply:
Don't think it matters really, `dist` is cool with me